readthedocsext/theme/templates/projects/pull_requests_form.html

```diff
@@ -0,0 +1,31 @@
+{% extends "projects/project_edit_base.html" %}
+
+{% load i18n %}
+{% load static %}
+{% load crispy_forms_filters %}
+
+{% block title %}{% trans "Pull request builds" %}{% endblock %}
+
+{% block project_pull_requests_active %}active{% endblock %}
+{% block project_edit_content_header %}{% trans "Pull request builds" %}{% endblock %}
+
+{% block project_edit_content %}
+  <p>
+    {% blocktrans trimmed %}
+    Enable builds for your pull requests and easily preview changes to your documentation.
+    {% endblocktrans %}
+  </p>
+
+  <form class="ui form" method="post" action=".">
```

Comment:
This isn't working correctly at the Form level, you can likely fix this at the form mixin class though. To show the form as disabled you need to specify the state in the template:

```suggestion
  <form class="ui {% if form.is_disabled %}disabled{% endif %} form" method="post" action=".">
```

Reply:
I removed this, since it made the link not clickable, and just made the input disabled 👍